Documentation.md

```diff
@@ -2148,6 +2148,112 @@ To use the GraphQL API, call the URL below with the Authorization header contain
 }
 </pre>
 
+## Resources <span hidden>GraphQL</span>
+
+### Satisfaction survey responses
+
+List all responses to the satisfaction survey during the given time interval.
+
+#### Query:
+
+| Parameter | Description | Values | Mandatory |
+| --- | --- | --- | --- |
+| interval | Filter responses on this time interval between 'from' and 'to' using ISO 8601 format | Object | Yes |
+| interval.from | Time interval start | ISO 8601 Date as String | Yes |
+| interval.to | Time interval end | ISO 8601 Date as String | Yes |
+| projectIds | Filter responses on project id | Array of Integer | No |
+| cursor | 'endCursor' of the previous request to get the next 100 responses | String | No |
+
+<pre class="prettyprint lang-js">
+query {
+    satisfactionSurveyResponses(
+        projectIds: [1,2], 
+        interval: {
+	    from: "2019-01-04T00:00:00+01:00",
+	    to: "2019-01-04T23:59:59+01:00"
+        }
+    ) {
+    pageData {
+      satisfactionSurveyResponse {
+        customerSatisfaction,
+        netPromoterScore,
+        satisfactionComment,
+        conversation {
+          id,
+          createdAt,
+          closedAt,
+          project {
+            id,
+            name,
+          },
+          agents {
+            id,
+            firstName,
+            lastName,
+            type
+          }
+        }
+      }
+    },
+    pageInfo {
+      endCursor
+      hasNextPage
+    }
+  }
+}
+</pre>
+
+#### Result:
+
+##### Paging
+'pageData' is an array of the 100 first responses.
+
+In 'pageInfo' the property 'hasNextPage' is true if there is more responses, and you can use the cursor 'endCursor' to fetch next responses.
```

Comment:
mettre ces infos dans le tableau et peut etre simplifié. Faut éviter le style direct `you can use endCursor` -> `endCursor can be used`